client/src/sentiment.js

```diff
@@ -0,0 +1,170 @@
+const fs = require('fs');
+const Sentiment = require('sentiment');
+
+const { dataStoreFilename } = require('./constant');
+const { removeHtmlTags } = require('./parse-utils');
+
+const storeData = require(dataStoreFilename);
+
+const range = (n) => Array(n).fill(undefined).map((_, i) => i);
```

Comment:
Gives an array like `[0, 1, 2, 3, 4, ...]`
You can't do `Array(n).map(...)` because the Array constructor yields a sparse array, and `map` works with elements, not indices.